free-programming-books.md

```diff
@@ -1082,6 +1087,7 @@ Original Source: [List of freely available programming books](http://web.archive
 * [Learn CSS Layout](http://learnlayout.com/)
 * [Scalable and Modular Architecture for CSS](http://smacss.com) - Jonathan Snook
 * [Web Audio API](http://chimera.labs.oreilly.com/books/1234000001552) - Boris Smus
+* [What Is HTML5?](http://shop.oreilly.com/product/0636920021049.do) - Brett McLaughlin
 * [Adaptive Web Design] (http://adaptivewebdesign.info/1st-edition/) - Aaron Gustafson
```

Comment:
I think it would be a good idea if someone would write a format validator script (python,ruby,etc) to take care of this before pull request. It would take care of common problems like spaces , `] (` and order all sections accordingly. And any other format problem @vhf would like to add.